src/app/contributors/contributors-loader.component.ts

```diff
@@ -0,0 +1,38 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+@Component({
+  selector: 'app-contributors-loader',
+  template: `
+    <div class="loader-container">
+      <div class="loader"></div>
+      <div class="loader"></div>
+      <div class="loader"></div>
+      <div class="loader"></div>
+      <div class="loader"></div>
+    </div>
+  `,
+  standalone: true,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  styles: [
+    `
+      .loader-container {
+        display: flex;
+        align-items: center;
+        gap: 8px;
+        .loader {
+          width: 32px;
+          height: 32px;
+          border-radius: 9999px;
```

Comment:
```suggestion
          border-radius: 4rem;
```
:-) is it not too much ???

Reply:
it's the tailwind approach for rounded elements. 50%, 100% , 9999px are same. 😁